susemanager-sync-data/susemanager-sync-data.changes

```diff
@@ -1,3 +1,4 @@
+- Add support for Alibaba Cloud Linux 2
```

Comment:
And this changes entry should go away as well, when we change additional_products.json later.

Reply:
Removed